### DISCUSSION TITLE
How is main revolt chat instance funded? Is it possible to donate?

### DISCUSSION THREAD

**Discussion by clarjon1** [stoatchat/discussions#574] [Q&A] [answered]
Heyo! Was looking thru the web site and the Q&A and couldn't find this info so thought i'd ask here:

Given other projects which I've used which start out free, but then start moving to trying to find business models, is there any way that users can donate to the team here to help make the main instance a bit easier to run? 

If not, is it possible to know about what sort of entity is providing the funds? Like a transparency report kinda thing?

Thanks in advance!

**Comment by Rexogamer** [comment, marked as answer]
You can donate to Insert [here](https://insrt.uk/donate) - if you donate £3 or more, you'll also be eligible for the Supporter badge. We don't currently produce transparency reports, but from what I know all funding has come from our own pockets and donations. cc @insertish

**Reply by clarjon1** [reply]
Thank you so much!

Of course, after getting this response, I finally noticed the "Donate to Revolt" button  in the app welcome screen. I didn't think to look in the app for that, d'oh.

Thank you for your time in responding! I've tossed a few bucks Insert's way ^_^
